packages/controllers/src/services/iframe/IframeExecutionService.ts

```diff
@@ -133,10 +133,11 @@ export class IframeExecutionService extends AbstractExecutionService<EnvMetadata
           resolve(iframe.contentWindow);
         }
       });
-      document.body.appendChild(iframe);
```

Comment:
The claim is that by appending the iframe to the DOM before setting its attributes, it would emit `load` before its contents were actually loaded?

Reply:
Yes. It seems that before it would be fired twice, but the promise would resolve the first time before the JS was actually loaded.

Reply:
It's hard to debug though because the events don't always seem to fire when stepping through it. But if I disable my breakpoints it works